lib/rubocop/cop/rspec/factory_girl/create_list.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryGirl
+        # Prefer using create_list over n.times { create :obj } calls.
+        #
+        # @example
+        #   # bad
+        #   3.times { create :user }
+        #
+        #   # good
+        #   create_list :user, 3
+        #
+        #   # good
+        #   3.times { |n| create :user, created_at: n.months.ago }
+        class CreateList < Cop
+          MSG = 'Prefer create_list.'.freeze
+
+          def_node_matcher :n_times, '(send (int $_) :times)'
+
+          def_node_matcher :times_block?, <<-PATTERN
+          (block
+            n_times
+            ...
+          )
+          PATTERN
+
+          def_node_matcher :factory_call?, <<-PATTERN
+            (send ${(const nil :FactoryGirl) nil} :create (sym $_) $...)
+          PATTERN
+
+          def on_block(node)
+            receiver, args, body = *node
```

Comment:
Should we check that this is a `times` block? I guess that’s what `times_block?` was meant for…